sparfa_server/tasks/loaders.py

```diff
@@ -43,17 +54,46 @@ def load_ecosystems_task():
     ecosystem_uuids = fetch_pending_ecosystems()
 
     if ecosystem_uuids:
-        results = group(load_ecosystem_task.s(ecosystem_uuid) for ecosystem_uuid in ecosystem_uuids)
+        results = group(load_ecosystem_task.si(ecosystem_uuid) for ecosystem_uuid in ecosystem_uuids)
         return results.apply_async(queue='beat-one')
 
 
 @celery.task
-def load_courses_task():
-    __logs__.info('starting up load courses task')
-    api_course_uuids = fetch_course_uuids()
-    __logs__.info('shoulve fetched something')
+def load_courses_events_task(course_events_requests):
+    if len(course_events_requests):
+        __logs__.info('Loading course events')
+        __logs__.info(course_events_requests)
+        next_course_events_requests = load_courses(course_events_requests)
+
+        if len(next_course_events_requests):
```

Comment:
len function is not needed. this can simply be `if next_course_events_requests:` etc.